xsuite/src/world/csworld.test.ts

```diff
@@ -0,0 +1,801 @@
+import { beforeEach, expect, test, beforeAll, afterAll, assert } from "vitest";
+import { assertAccount, assertVs } from "../assert";
+import { e } from "../data";
+import { Tx } from "../proxy";
+import { DummySigner } from "./signer";
+import {
+  generateContractU8AAddress,
+  generateWalletU8AAddress,
+  isContractAddress,
+} from "./utils";
+import { CSWorld, CSContract, CSWallet } from ".";
+
+let world: CSWorld;
+let wallet: CSWallet;
+let otherWallet: CSWallet;
+let contract: CSContract;
+const fftId = "FFT-abcdef";
+const sftId = "SFT-abcdef";
+const worldCode = "file:contracts/world/output/world.wasm";
+const zeroBechAddress =
+  "erd1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6gq4hu";
+const zeroHexAddress =
+  "0000000000000000000000000000000000000000000000000000000000000000";
+const zeroBytesAddress = new Uint8Array(32);
+const emptyAccount = {
+  nonce: 0,
+  balance: 0,
+  code: "",
+  codeHash: "",
+  codeMetadata: "",
+  owner: "",
+  kvs: {},
+};
+const explorerUrl = "http://explorer.local";
+
+beforeAll(async () => {
+  world = await CSWorld.start({
+    explorerUrl,
+    // verbose: true,
+    // debug: true,
+    waitFor: 120_000,
```

Comment:
Tests should pass without `waitFor`.

Reply:
The reason this was here was because when updating chain simulator version we need to delete the configs, and then next time the Chain Simulator is started it will need to download them, and that happened in tests since that was easier for me.

Although I will remove this since the Chain Simulator binary can be started manually after we update the version.